backbone.js

```diff
@@ -1314,28 +1296,24 @@
 
     // jQuery delegate for element lookup, scoped to DOM elements within the
     // current view. This should be preferred to global lookups where possible.
-    $: function(selector) {
+    $: function (selector) {
       return this.$el.find(selector);
     },
 
-    // preinitialize is an empty function by default. You can override it with a function
-    // or object.  preinitialize will run before any instantiation logic is run in the View
-    preinitialize: function(){},
-
     // Initialize is an empty function by default. Override it with your own
     // initialization logic.
-    initialize: function(){},
+    initialize: function () {},
 
     // **render** is the core function that your view should override, in order
     // to populate its element (`this.el`), with the appropriate HTML. The
     // convention is for **render** to always return `this`.
-    render: function() {
+    render: function () {
       return this;
     },
 
     // Remove this view by taking the element out of the DOM, and removing any
     // applicable Backbone.Events listeners.
-    remove: function() {
+    remove: function () {
```

Comment:
I think this rule should change.  Linting is great, but there's no need to modify the current style if there is already a precedent.

Reply:
Please edit the .eslintrc with the desired spec. My primary concern is bug-hunting, followed by figuring out how best to make sure this & underscore are working, so I know my efforts on floorplan aren't in vain.